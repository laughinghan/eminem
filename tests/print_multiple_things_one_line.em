let .x 1
let .y "lol"
print "x: " x ", y:" y "
"
