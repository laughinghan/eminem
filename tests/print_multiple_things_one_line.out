x: 1, y:lol
